File: zenwave-jpa-example/src/main/java/io/zenwave360/example/core/inbound/dtos/CustomerSearchCriteria.java
package io.zenwave360.example.core.inbound.dtos;

import java.io.Serializable;

/** CustomerSearchCriteria. */
@lombok.Getter
@lombok.Setter
public class CustomerSearchCriteria implements Serializable {

    private String name;

    private String email;

    private String city;

    private String state;

}

File: examples/kustomer-address-jpa/src/main/kotlin/io/zenwave360/examples/kotlin/service/dtos/CustomerSearchCriteria.kt
package io.zenwave360.examples.kotlin.service.dtos

import io.zenwave360.examples.kotlin.domain.*
import java.io.Serializable
import java.math.*
import java.time.*
import jakarta.validation.constraints.*

/**
 * CustomerSearchCriteria.
 */
data class CustomerSearchCriteria(

    
    val name: String?  = null,

    
    val email: String?  = null,

    
    val city: String?  = null,

    
    val state: String?  = null

)  : Serializable {






}
